spec/ace.md

```diff
@@ -383,6 +383,33 @@ Small quantities can be represented directly as decimals (e.g., 0.3), or using m
 
 **NOTE**: Network limits MUST NOT apply to localhost communication between apps in a pod.
 
+### Environment Isolators
+
+An application or a group of applications may often require some tweaking to the environment where they run, in order to perform in an optimal way.
+Environment isolators take care of setting up the runtime environment as needed.
+
+#### environment/sysctl
+
+* Scope: pod
+
+**Parameters:**
+
+* dictionary of `sysctl(8)` key-value entries to set kernel runtime parameters.
+
+```json
+"name": "environment/sysctl",
+"value": {
+  "sysctl": {
+    "net.ipv4.tcp_mtu_probing": "1",
+    "net.ipv4.tcp_keepalive_time": "300",
+    "net.ipv4.tcp_keepalive_probes": "5",
+    "net.ipv4.tcp_keepalive_intvl": "15",
+  }
+}
+```
+**Notes**:
+ 1. Only a single `environment/sysctl` isolator can be specified per-pod.
+ 2. Implementations MAY sanity-check keys before applying sysctl parameters. For example, only entries related to isolated subsystems (e.g. `net.*` on Linux) may be allowed.
```

Comment:
"will not impact other pods" is a very fuzzy term (compare https://github.com/kubernetes/kubernetes/pull/26057/files#diff-8dc23ab258695ee42154d4d1238c36efR323). Being namespaced in certain namespaces is much clearer defined (might differ by kernel version though).

Also note, that being namespaced of course depends on which namespaces are actually used (ipc, net, etc.) and not shared with the host. This should have influence on the allowed sysctls.